sdk/go/common/resource/plugin/plugin.go

```diff
@@ -635,15 +635,13 @@ func (p *plugin) Close() error {
 		//	To assist with debugging we have dumped the STDOUT and STDERR streams of the plugin:
 		//
 		//	<output>
-		d.Errorf(diag.StreamMessage("", fmt.Sprintf("\n\n         Detected that %s exited prematurely.\n", p.Bin), id))
-		d.Errorf(diag.StreamMessage("",
-			"         This is *always* a bug in the provider. "+
-				"Please report the issue to the provider author as appropriate.\n\n", id))
-		d.Errorf(diag.StreamMessage("",
-			"To assist with debugging we have dumped the STDOUT and STDERR streams of the plugin:\n\n", id))
 		p.unstructuredOutput.outputLock.Lock()
 		defer p.unstructuredOutput.outputLock.Unlock()
-		d.Errorf(diag.StreamMessage("", p.unstructuredOutput.output.String(), id))
+		d.Errorf(diag.StreamMessage("",
+			fmt.Sprintf("Detected that %s exited prematurely.", p.Bin), id))
+		d.Errorf(diag.StreamMessage("", fmt.Sprintf("This is *always* a bug in the provider. "+
```

Comment:
Could this be just one StreamMessage?

Reply:
And just put these two things on one line?  I think I'd be good with that, will change.

Reply:
Actually, I think that would look a little worse:
![image](https://github.com/user-attachments/assets/d9d0c82e-8882-43f5-8538-e80369424e87)

With a newline instead:
![image](https://github.com/user-attachments/assets/11ca4e03-eb6f-462e-aa0c-cf9782393cc8)

I think what I have currently looks best, but I'm also okay with the one with a newline here.

Reply:
Because I left the "This is always a bug" line in a separate `d.Errorf`.  We can also pack that into the first message. (Note that the error message has the `error:` already embedded, so there's no straightforward way to just remove that.

Reply:
yeh that looks fine